sdf.md

```diff
@@ -1816,10 +1818,72 @@ The initial set of registrations is described in {{sdftype-r}}.
 Security Considerations {#seccons}
 =======================
 
-Some wider issues are discussed in {{-seccons}}.
-
-(Specifics: TBD.)
-
+Some wider security considerations applicable to Things are discussed
+in {{-seccons}}.
+{{Section 5 of -cddl}} gives an overview over security considerations
+that arise when formal description techniques are used to govern
+interoperability; analogs of these security considerations can apply
+to SDF.
+
+The security considerations of underlying building blocks such as
+those detailed in {{Section 10 of -utf8}} apply.
+SDF uses JSON as a representation language; {{-json}} for a number of
+cases indicates that implementation behavior for certain constructs
+allowed by the JSON grammar is unpredictable.
+Implementations need to be robust against invalid or unpredictable
+cases on input, preferably by rejecting input that is invalid or
+that would lead to unpredictable behavior, and need to avoid generating
+these cases on output.
+
+Implementations of model languages may also exhibit
+performance-related availability issues when the attacker can control
+the input, see {{Section 4.1 of -jsonpath}} for a brief discussion.
+
+SDF may be used in two processes that are often security relevant:
+*validation* of data that is intended to be described by SDF models, and
+*augmentation* of these data with information obtained from the SDF
+models that apply.
+
+Implementations need to ascertain the provenance and applicability of
+the SDF models they employ operationally in these ways.
```

Comment:
Where "these ways" refers to would be more clear without the paragraph break above

Reply:
I think the paragraph break is useful, so I changed "these ways" into " such security relevant ways".